wire-ios-sync-engine/Source/Synchronization/ZMHotFixDirectory.m

```diff
@@ -74,11 +74,6 @@ - (NSArray *)patches
                      patchCode:^(NSManagedObjectContext *context) {
                          [ZMHotFixDirectory insertNewConversationSystemMessage:context];
                      }],
-                    [ZMHotFixPatch
-                     patchWithVersion:@"45.1"
-                     patchCode:^(NSManagedObjectContext *context) {
-                         [ZMHotFixDirectory updateSystemMessages:context];
-                     }],
```

Comment:
We shouldn't be removing hot fixes like this unless we're sure that this version isn't existing in the wild anymore. It's such an old version that it's unlikely to still exist, but it would be good to find out which wire version contains this version.

Also, if we can remove this hot fix, then we can remove all previous hot fixes.

Reply:
- figure out which version of the app contained SE 45.1, check the git tags
- figure out what the earliest non blacklisted client version is, you can check the blacklist for this.